README.md

```diff
@@ -122,44 +122,43 @@ SIG-specific GitHub discussions.
+Java (SDK + Instrumentation)|Every Thursday at 09:00 PT| [opentelemetry-calendar-java@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-java)                             | [Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/)                            | [#otel-java](https://cloud-native.slack.com/archives/C014L2KCTE3) and [#otel-java-instrumentation](https://cloud-native.slack.com/archives/C0150QF88FL)      |
+JavaScript: SDK|Every Wednesday at 09:00 PT| [opentelemetry-calendar-js@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-js)                                 | [Google Doc](https://docs.google.com/document/d/1YR1Z3SqOkAJGffycO_P9d0CiTMP79OOtu_xvIyyVLME/edit?usp=sharing)            | [#otel-js](https://cloud-native.slack.com/archives/C01NL1GRPQR) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-js/discussions)     |
+PHP: SDK|Every Wednesday at 10:30 PT| [opentelemetry-calendar-php@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-php)                               | [Google Doc](https://docs.google.com/document/d/1WLDZGLY24rk5fRudjdQAcx_u81ZQWCF3zxiNT-sz7DI/edit)                        | [#otel-php](https://cloud-native.slack.com/archives/C01NFPCV44V)                                                                                             |
+Python: SDK|Every Thursday at 09:00 PT| [opentelemetry-calendar-python@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-python)                         | [Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)                        | [#otel-python](https://cloud-native.slack.com/archives/C01PD4HUVBL)                                                                                          |
+Ruby: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-ruby@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-ruby)                             | [Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)                        | [#otel-ruby](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions) |
+Rust: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |                                                                                                 |
```

Comment:
You could add their Slack channel as well (see @hdost's PR #1418):
```suggestion
Rust: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |[#otel-rust](https://cloud-native.slack.com/archives/C03GDP0H023) and [Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
```

Reply:
@trask if you want to cherry pick my change that would be preferable.

Reply:
@hdost I left a couple comments on your PR to clarify